contracts/MyStrategy.sol

```diff
@@ -132,6 +137,11 @@ contract MyStrategy is BaseStrategy, ReentrancyGuardUpgradeable {
         }
     }
 
+    function setAuraBalToBalEthBptMinOutBps(uint256 _minOutBps) external {
+        _onlyGovernanceOrStrategist();
```

Comment:
I think we should add a require to ensure that the mount is never above MAX_BPS, otherwise harvests will fail.